de/nmichael/efa/data/storage/DataFile.java

```diff
@@ -241,8 +242,10 @@ public synchronized void openStorageObject() throws EfaException {
             if (recovered || shouldWriteMirrorFile()) {
                 saveStorageObject();
             }
-            if ((Daten.project != null) && (Daten.project.getProjectStorageType() == IDataAccess.TYPE_EFA_CLOUD))
-                Ecrid.addAll(this.getPersistence());
+            String fName = filename.substring(filename.lastIndexOf('.') + 1);
+            if (TableBuilder.tablenames.contains(fName))
```

Comment:
this only works if 
- fName does not contain a path
- Also, fName does not contain uppercase letters.
  - Suggestion:   String fName = filename.substring(filename.lastIndexOf('.') + 1).toLower();